.github/workflows/ci.yml

```diff
@@ -82,3 +82,26 @@ jobs:
     - uses: actions-rs/cargo@v1
       with:
         command: build
+
+  integration:
```

Comment:
I mean you should write integration tests like that - https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html

For example: Create the directory `tests` and test the function `run` from `lib.rs`.

Reply:
@mstruebing I've thought, probably I should write integration tests instead of you because I have some ideas. Don't you mind?

Reply:
Yes sure, sorry for not pushing this forward :)